cdn/dovetail-cdn.yml

```diff
@@ -13,53 +13,110 @@ Metadata:
       OriginBucket:
-        default: Primary dovetail-cdn-arranger S3 Bucket
+        default: Primary region's dovetail-cdn-arranger S3 Bucket
+      OriginBucket2:
+        default: Secondary region's dovetail-cdn-arranger S3 Bucket
+      OriginBucket3:
+        default: Tertiary region's dovetail-cdn-arranger S3 Bucket
       DistributionDomain:
         default: Domain Name
       ExpiredRedirectPrefix:
         default: Prefix for redirecting back to Dovetail
       OriginRequestFunctionArn:
         default: Dovetail origin request lambda
-      EnvironmentTypeAbbreviation:
-        default: Environment type used in naming things
+      EnvironmentType:
+        default: Environment Type
+      CacheBehaviorPrefix:
+        default: Primary region's path prefix
+      CacheBehaviorPrefix2:
+        default: Secondary region's path prefix
+      CacheBehaviorPrefix3:
+        default: Tertiary region's path prefix
       RealtimeLogKinesisStreamArn:
-        default: Kinesis stream for realtime logs
+        default: Primary region's kinesis stream for realtime logs
+      RealtimeLogKinesisStreamArn2:
+        default: Secondary region's kinesis stream for realtime logs
+      RealtimeLogKinesisStreamArn3:
+        default: Tertiary region's kinesis stream for realtime logs
+      RealtimeLogFields:
+        default: Fields to include in realtime logs
 
 Parameters:
   OriginBucket:
     Type: String
-    Description: eg. some-bucket-name.s3.amazonaws.com
+    Description: eg. some-bucket-name.s3.region-name.amazonaws.com
+  OriginBucket2:
+    Type: String
+    Description: eg. some-bucket-name.s3.region-name.amazonaws.com
+  OriginBucket3:
+    Type: String
+    Description: eg. some-bucket-name.s3.region-name.amazonaws.com
   DistributionDomain:
     Type: String
     Description: eg. dovetail3-cdn.prxu.org
   ExpiredRedirectPrefix:
     Type: String
-    Description: eg. https://dovetail.prxu.org/_
+    Description: eg. https://dovetail.prxu.org
   OriginRequestFunctionArn:
     Type: String
     Description: eg. arn:aws:lambda:<region>:<account>:function:<name>:<version>
   EnvironmentType:
     Type: String
     AllowedValues:
+      - Testing
       - Staging
       - Production
-  EnvironmentTypeAbbreviation:
+  CacheBehaviorPrefix:
     Type: String
-    AllowedValues:
-      - test
-      - stag
-      - prod
+    Description: eg. /use1/*, /us-west-2/*
+  CacheBehaviorPrefix2:
+    Type: String
+    Description: eg. /use1/*, /us-west-2/*
+  CacheBehaviorPrefix3:
+    Type: String
+    Description: eg. /use1/*, /us-west-2/*
   RealtimeLogKinesisStreamArn:
     Type: String
     Description: eg. arn:aws:kinesis:<region>:<account>:stream/<name>
+  RealtimeLogKinesisStreamArn2:
+    Type: String
+    Description: eg. arn:aws:kinesis:<region>:<account>:stream/<name>
+  RealtimeLogKinesisStreamArn3:
+    Type: String
+    Description: eg. arn:aws:kinesis:<region>:<account>:stream/<name>
+  RealtimeLogFields:
+    Type: String
```

Comment:
If this is a `CommaDelimitedList`, you get the `Split` for free

Reply:
Ooh, didn't know that was a thing.